tests/test_tacacs_global.rb

```diff
@@ -72,13 +72,15 @@ def test_tacacs_global
     assert_equal(7, global.key_format)
     assert_equal('"WAWY_NZB"', global.key)
 
-    # second key change - modify key to type6
-    key_format = 6
-    # Must use a valid type6 password: CSCvb36266
-    key = 'JDYkqyIFWeBvzpljSfWmRZrmRSRE8'
-    global.encryption_key_set(key_format, key)
-    assert_equal(key_format, global.key_format)
-    assert_equal("\"#{key}\"", global.key)
+    unless Platform.image_version[/I2/]
```

Comment:
Do we have a defect we can reference here and call  the `skip_legacy_method` instead for I2 images?

Reply:
I will check again, I could not find any defect on topic. But even if we, how do you call skip_legacy_defect here? We want all the test code to run just above these 4 lines and all the code below as well. Skip simply gets out of the test right away, right?

Reply:
We would need to pull this out into a separate test.  Are we certain this is only a problem on I2?  If so then it's probably ok like it is.  We should be moving away from I2 image validation soon.

Reply:
I am leaving this test as it is, since I2 is going away soon. No point in writing separate tests just for a short time. In the meantime it will fail if it runs against I2 but that's fine

Reply:
I did not mean to remove the I2 check.  I still think that is good.  No point in always clicking on the email just to see that it's the same problem over and over again.  If we are not going to fix it then we should leave the I2 check.  I only meant that we don't need to separate this out into it's own test.

Reply:
Ok added it back. I got confused by your comment. Sorry!